apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx

```diff
@@ -44,9 +45,15 @@ interface ManagerDashboardProps {
   userQuery: FragmentType<typeof ManagerDashboardUser_Fragment>;
 }
 
+type AccordionItems = ("your_talent_searches" | "")[];
+
 const ManagerDashboard = ({ userQuery }: ManagerDashboardProps) => {
   const paths = useRoutes();
   const intl = useIntl();
+  const [accordionItems, setAccordionItems] = useLocalStorage(
```

Comment:
You can tighten the typing up a bit here.
```suggestion
  const [accordionItems, setAccordionItems] = useLocalStorage<AccordionItems>(
```